tests/pages/services/ServiceActions-cy.js

```diff
@@ -37,6 +38,22 @@ describe("Service Actions", function() {
           expect($menuItem.length).to.equal(0);
         });
     });
+
+    it('does not display the "Open Service" option for SDK services', function() {
```

Comment:
Just added a text case specifically for SDK services, and slight refactor removing the `beforeEach()`.